### PR TITLE
Update cluster configuration to increase worker memory and remove ray…

### PR DESCRIPTION
**Pull request by abhijeet-dhumal**
# Issue link

[RHOAIENG-9504](https://issues.redhat.com/browse/RHOAIENG-9504)
Verifies - https://github.com/project-codeflare/codeflare-sdk/pull/582
# What changes have been made

- Updated cluster configuration to increase worker memory
- Removed ray-image input parameter given in cluster configuration for e2e tests.

# Verification steps

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

**Review by Bobbins228 — COMMENTED** [review]
/lgtm 
@abhijeet-dhumal just wondering for my own sake why we need to increase the worker memory?

**Comment by abhijeet-dhumal** [comment]
> /lgtm @abhijeet-dhumal just wondering for my own sake why we need to increase the worker memory?

Hey @Bobbins228 Thank you for reviewing! 
Actually while running tests with 2Gb worker memory the tests were failing with error 
`memory used=0.09GB) was running was 1.77GB / 1.86GB (0.951319)` where increasing worker memory worked in this case.  Iirc we had a conversation recently through DM regarding same scenario. 

image (98)

File: tests/e2e/local_interactive_sdk_oauth_test.py [hunk 1]
```diff
@@ -46,9 +44,8 @@ def run_local_interactives(self):
                 num_workers=1,
                 worker_cpu_requests=1,
                 worker_cpu_limits=1,
-                worker_memory_requests=4,
+                worker_memory_requests=1,
```
Comment (sutaakar) [inline tests/e2e/local_interactive_sdk_oauth_test.py:47]: Is this correct change?

Comment (abhijeet-dhumal) [reply tests/e2e/local_interactive_sdk_oauth_test.py:47]: @sutaakar Yes, I have tested this configuration with 1 memory requests for local-interactive test, worked seamlessly! 

image

Comment (sutaakar) [reply tests/e2e/local_interactive_sdk_oauth_test.py:47]: Oh, ok
I though it was a limit. Thanks

**Review by Bobbins228 — APPROVED** [review]
/approve

**Pull request merged**